lib/pure/collections/critbits.nim

```diff
@@ -156,7 +156,7 @@ proc excl*[T](c: var CritBitTree[T], key: string) =
   discard exclImpl(c, key)
 
 proc missingOrExcl*[T](c: var CritBitTree[T], key: string): bool =
-  ## Returns true iff `c` does not contain the given `key`. If the key
+  ## Returns true if `c` does not contain the given `key`. If the key
```

Comment:
I really like the clarity of "if and only if", maybe we could spell it out explicitly in the docstrings?

There are many instances where the following sentence contains a sentence with "if not" semantics. If the first sentence says "if and only if" it becomes clear that the second sentence is only a refining information.

E.g. here if we only read the weaker first sentence:

> Returns true if `c` does not contain the given `key`.

and then 

> If the key does exist, c.excl(key) is performed.

the question raises what is returned in that negated case? Can it be `true` as well? Maybe that depends on whether `c.excl(key)` is successful or so?

Reading a strong "if and only if" first avoids that.

Reply:
Returning implies there is no need for "and only if", the proc is done. "If and only if the key does exist, c.excl(key) is performed" makes sense, though it's painful to read

Reply:
I think these questions are answered more simply:

> Returns true if `c` contains the given `key`.  Otherwise, equivalent to ``c[key] = val``.

Programmers are reading this.  Lets give them a mental model of the control-flow.